src/gui/qgsattributedialog.cpp

```diff
@@ -66,15 +66,19 @@ void QgsAttributeDialog::setHighlight( QgsHighlight *h )
 
 void QgsAttributeDialog::accept()
 {
-  bool didSave = mAttributeForm->save();
+  QString error;
+  bool didSave = mAttributeForm->saveWithDetails( &error );
```

Comment:
```suggestion
  const bool didSave = mAttributeForm->saveWithDetails( &error );
```

Reply:
Every now and then I checkout https://reviews.llvm.org/D54943 and hope that it's finally been merged... I'd love to run that tidy over our whole codebase and const correct everything :smile: